Mathlib/Data/MvPolynomial/Basic.lean

```diff
@@ -1215,11 +1215,11 @@ theorem eval₂_eq_eval_map (g : σ → S₁) (p : MvPolynomial σ R) : p.eval
     simp only [comp_apply, eval₂_X]
 #align mv_polynomial.eval₂_eq_eval_map MvPolynomial.eval₂_eq_eval_map
 
--- Porting note: new theorem
 -- This probably belongs earlier, but it breaks the fragile proof of `eval₂_eq_eval_map`
 @[simp]
 theorem eval₂_id (p : MvPolynomial σ R) : eval₂ (RingHom.id _) g p = eval g p :=
```

Comment:
This lemma is right before `eval_eval₂` in Lean 3; can you move it, or add a porting note saying that it moved?

Reply:
Porting note made clearer.